### DISCUSSION TITLE
关于plot3d -W+Ve0.3i的疑问

### DISCUSSION THREAD

**Discussion by A1b8** [gmt-china/GMT_docs#1106] [Q&A] [answered]
大家好，
我正在尝试利用plot3d绘制3D坐标系下的线条属性，下面是我测试的代码。
cat << EOF > test.txt
5 0 0
5 5 5
EOF
gmt begin test1
gmt basemap -R0/5/0/5/0/5 -JX5c -JZ5c -Bxa1fg -Bya1fg -Bza1fg+l"Z" -BwSEnZ -p20/20
gmt plot3d test.txt -W1.5p+ve0.2i+gblue -p
gmt end show
为什么线条属性'-W1.5p+ve0.2i+gblue'在3D坐标系下无法正确显示箭头属性？（2D坐标系下能正常显示出箭头属性）。
提前感谢能为我解惑的老师同学！！！

**Comment by seisman** [comment, marked as answer]
感觉是3D下的bug
